src/main/java/com/jobs/portal/jobsportal/request/JobPostRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.jobs.portal.jobsportal.request;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.sql.Timestamp;
+
+@Data
+public class JobPostRequest extends BaseRequest{
+    @NotBlank
+    private String username;
+    @NotBlank
+    private String title;
+    private String description;
+    @NotBlank
+    private String country;
+    @NotBlank
+    private String city;
+    @NotBlank
+    private String companyName;
+    private String companyAddress;
+    @NotBlank
+    private String experienceStart;
+    private String experienceEnd;
+    private String salaryStart;
+    private String salaryEnd;
+    @NotBlank
+    private String qualification;
+    private String totalVacancies;
+    @NotNull
+    private Timestamp postedOn;
```

Comment:
postedOn will not be part of request, It will be current time at time of saving entity

Reply:
Done